pkgs/tools/security/rustscan/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, makeWrapper
+, nmap
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rustscan";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "RustScan";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "0rkqsh4i58cf18ad97yr4f68s5jg6z0ybz4bw8607lz7cjkfvjay";
+  };
+
+  cargoSha256 = "0mj214f2md7kjknmcayc5dcfmlk2b8mqkn7kxzdis8qv9a5xcbk8";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  fixupPhase = ''
+    wrapProgram $out/bin/rustscan --prefix PATH ":" ${nmap}/bin ;
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Faster Nmap Scanning with Rust";
+    homepage = https://github.com/RustScan/RustScan;
```

Unquoted URLs are deprecated.

```suggestion
    homepage = "https://github.com/RustScan/RustScan";
```